docs/en/ingest-management/release-notes/release-notes-8.4.asciidoc

```diff
@@ -0,0 +1,235 @@
+// Use these for links to issue and pulls.
+:kib-issue: https://github.com/elastic/kibana/issues/
+:kibana-pull: https://github.com/elastic/kibana/pull/
+:agent-issue: https://github.com/elastic/elastic-agent/issues/
+:beats-issue: https://github.com/elastic/beats/issues/
+:agent-libs-pull: https://github.com/elastic/elastic-agent-libs/pull/
+:agent-pull: https://github.com/elastic/elastic-agent/pull/
+:fleet-server-issue: https://github.com/elastic/beats/issues/fleet-server/
+:fleet-server-pull: https://github.com/elastic/beats/pull/fleet-server/
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-8.4.0>>
+
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 8.4.0 relnotes
+
+[[release-notes-8.4.0]]
+== {fleet} and {agent} 8.4.0
+
+Review important information about the {fleet} and {agent} 8.4.0 release.
+
+[discrete]
+[[breaking-changes-8.4.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and
+performance. Before you upgrade, review the breaking changes, then mitigate the
+impact to your application.
+
+[discrete]
+[[breaking-135669]]
+.xpack.agents.* are uneditable in UI when defined in kibana.yml
+[%collapsible]
+====
+*Details* +
+When you configure `setxpack.fleet.agents.fleet_server.hosts` and `xpack.fleet.agents.elasticsearch.hosts` in kibana.yml, you are unable to update the fields on the Fleet UI. 
+For more information, refer to {kibana-pull}135669[#135669].
+
+*Impact* +
+To configure `setxpack.fleet.agents.fleet_server.hosts` and `xpack.fleet.agents.elasticsearch.hosts` on the Fleet UI, avoid configuring the settings in kibana.yml.
+====
+
+[discrete]
+[[new-features-8.4.0]]
+=== New features
+
+The 8.4.0 release adds the following new and notable features. 
+ 
+{fleet}::
+* Enable package signature verification feature {kibana-pull}137239[#137239]
+* Allow user to force install an unverified package {kibana-pull}136108[#136108]
+* Display package verification status {kibana-pull}135928[#135928]
```

Comment:
Yeah, let's skip the release notes for the package verification items until it is turned on